### PR TITLE
adds list of Zelda characters by game

### PR DESCRIPTION
**Pull request by audy**
Source: http://zelda.neoseeker.com/wiki/Character_list

### CODE REVIEW

File: data/games/zelda.json [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+        "Link",
+        "Lord Jabu-Jabu",
+        "Maku Tree",
+        "Malon",
+        "Mamamu Yan",
+        "Maple",
+        "Moosh",
+        "Mrs. Ruul",
+        "Nayru",
+        "Nayru",
```
Comment (hugovk) [inline data/games/zelda.json]: "Nayru" is in the list twice, please could you remove one?

**Comment by audy** [comment]
Oops. Fixed. @hugovk

**Comment by dariusk** [comment]
Forgot to merge this. Thanks for the contribution!

**Comment by audy** [comment]
⚔️

**Pull request merged**